identity/identity.go

```diff
@@ -0,0 +1,77 @@
+package identity
+
+import (
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"fmt"
+)
+
+const PASSPHRASE = ""
+
+type IdentityManager struct {
+	keystoreManager *keystore.KeyStore
+}
+
+func NewIdentityManager(keydir string) *IdentityManager {
+	return &IdentityManager{
+		keystoreManager: keystore.NewKeyStore(keydir, keystore.StandardScryptN, keystore.StandardScryptP),
+	}
+}
+
+func (idm *IdentityManager) CreateNewIdentity() (string, error) {
+	account, err := idm.keystoreManager.NewAccount(PASSPHRASE)
+	if err != nil {
+		return "", err
+	}
+
+	return account.Address.Hex(), nil
+}
+
+func (idm *IdentityManager) GetIdentities() []string {
+	var ids []string
+	for _, account := range idm.keystoreManager.Accounts() {
+		ids = append(ids, account.Address.Hex())
+	}
+	return ids
+}
+
+func (idm *IdentityManager) IdentityExists() (bool) {
```

Comment:
IdentityExists method sounds like it should lookup for an Account in keystore by string.
it now returns true if at least one id was registered.